### PR TITLE
10207: add CDC elims metabase queries to source control

### PR DESCRIPTION
**Pull request by mkalish**
This PR adds the queries written for the CDC ELIMS team to source control

Test Steps:
1. Go to https://staging.prime.cdc.gov/metabase/collection/49-cdc-elims-collection and check out all the queries

**As a note @arnejduranovic  has already tested these as part of figuring out the functionality**

## Changes
- Adds the queries for the CDC ELIMS team

### Testing
- [x] Tested locally?
- [ ] Ran `./prime test` or `./gradlew testSmoke` against local Docker ReportStream container?
- [ ] (For Changes to /frontend-react/...) Ran `npm run lint:write`? 
- [ ] Added tests?

### Process
- [ ] Are there licensing issues with any new dependencies introduced?
- [ ] Includes a summary of what a code reviewer should test/verify?
- [ ] Updated the release notes?
- [ ] Database changes are submitted as a separate PR?
- [ ] DevOps team has been notified if PR requires ops support?

## Linked Issues
- #10207

### CODE REVIEW

File: prime-router/examples/cdc-elims-metabase-queries/README.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+# CDC ELIMS Metabase Queries
+
+These are source controlled copies of queries that we have provided to the CDC ELIMS as troubleshooting tools during the
+onboarding process.  They as exist as questions in both the staging and production instances of metabase in a
```
Comment (JFU-NAVA-PBC) [inline prime-router/examples/cdc-elims-metabase-queries/README.md:4]:
```suggestion
onboarding process.  They exist as questions in both the staging and production instances of metabase in a
```

File: prime-router/examples/cdc-elims-metabase-queries/active-stlts.sql [hunk 2]
```diff
@@ -0,0 +1,9 @@
+select  org.name || '.' || receiver.name as "Receiver Name", receiver.created_at as "Created At"
+from setting as receiver
+join setting org on org.setting_id = receiver.organization_id
+where receiver."type" = 'RECEIVER'
+and receiver."values" ->> 'topic' = 'elr-elims'
+and receiver.is_active = true
+and receiver."values" ? 'transport'
+and receiver.is_deleted = falsea
```
Comment (JFU-NAVA-PBC) [inline prime-router/examples/cdc-elims-metabase-queries/active-stlts.sql:8]:
```suggestion
and receiver.is_deleted = false
```

Comment (mkalish) [reply prime-router/examples/cdc-elims-metabase-queries/active-stlts.sql:8]: Thanks, fat finger from copying out of metabase!

**Comment by JFU-NAVA-PBC** [comment]
this is out of scope of this PR, just a fyi:

got syntax error when I ran after putting in some random parameter value:

image

**Comment by mkalish** [comment]
@JFU-NAVA-PBC Updated that query to work around a metabase bug, so this is good to go

**Review by JFU-NAVA-PBC — APPROVED** [review]
Looks good, probably it is a bit misleading to save these metabase scripts under examples folder, they are production assets, but this can be addressed later.

**Pull request merged**